libp2p/Host.cpp

```diff
@@ -749,46 +733,54 @@ void Host::startedWorking()
     // initialization (e.g. start capability threads, start TCP listener, and kick off timers)
     asserts(!m_timer);
```

Comment:
As we don't delete `m_timer` now, this assert will fail when we stop and then restart network